readme.md

```diff
@@ -9,29 +9,34 @@ workflows of an InVEST user.
 ## To develop and launch this Application
 
 * `npm install`  
-* clone natcap.invest and checkout a recent revision (e.g. `main`)  
-* setup a conda* environment with deps for `natcap.invest Flask PyInstaller`  
-* build invest binaries  
-	`python -m PyInstaller --workpath build/pyi-build --clean --distpath build ./invest-flask.spec`  
-* `npm start`  
+* bind to an `invest` executeable (see package.json "invest" for a compatible version)

```

Comment:
Related to this and `invest` in `package.json`, is this something that developers are expected to change frequently and commit in PRs?

Reply:
I don't think it will be changed frequenty, but yes, the "invest" property will need to be updated to whatever version of invest we want packaged with the workbench. Maybe in the future we would want to programmaticaly default to the latest invest release, but for now hardcoding the invest version in package.json seems okay to me.